packages/rsocket-flowable/src/FlowableProcessor.js

```diff
@@ -23,7 +22,7 @@ export default class FlowableProcessor<T, R>
 
   onNext(t: T) {
     if (!this._sink) {
-      warning('Warning, premature onNext for processor, dropping value');
+      console.warn('console.warn, premature onNext for processor, dropping value');
```

Comment:
I think this should be `'Warning, premature...'`, or just `premature onNext...` to align with the warning message below.

Reply:
Good point! I think I accidentally did a wrong find/replace here.

Reply:
Fixed in https://github.com/rsocket/rsocket-js/pull/134/commits/0c7e53a39264b3af802509d649f2f5b8d080f72c